.gitignore

```diff
@@ -3,29 +3,27 @@
 # dependencies
 */node_modules
 /.pnp
-.pnp.js
 
 # testing
 /coverage
-.coverage
 
 # production
 /build
 
 # misc
-.DS_Store
```

Comment:
How come you got rid of these

Reply:
he hates mac users :rofl: 

jk

Reply:
I offered using `.*` to grab all instances of hidden files instead of naming each one. My proposal still covers all of the existing ones and covers swap files too. Just a bit of a cleaner approach!

Reply:
Don't we want to keep .github from being ignored though?

Reply:
yeah, there's also `.gitignore` and .`coveragerc` that we shouldn't ignore

Reply:
The wildcard actually doesn't ignore .gitignore as you can see it hasn't been ignored in my commit but I didn't know about .coveragerc. Okay, no problem I'll change it back. No harm in asking :)